.github/workflows/publish.yml

```diff
@@ -30,12 +30,18 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.13'
+          python-version: '3.11.0'
```

Comment:
why are we going back to 3.11?

Reply:
3.12 and above does not contain disutils, referrence: https://stackoverflow.com/questions/77247893/modulenotfounderror-no-module-named-distutils-in-python-3-12